doc/user/content/ops/network-security/ssh-tunnel.md

```diff
@@ -51,6 +51,49 @@ CREATE CONNECTION ssh_connection TO SSH TUNNEL (
     echo "ssh-ed25519 AAAA...76RH materialize" >> ~/.ssh/authorized_keys
     ```
 
+1. Configure your internal firewall to allow the SSH bastion host to connect to your Kafka cluster or PostgreSQL instance.
+
+    If you are using a cloud provider like AWS or GCP, update the security group or firewall rules for your PostgreSQL instance or Kafka brokers.
+
+    You'll need to allow incoming traffic from the IP address of the SSH bastion host on the necessary ports.
+
+    For example, use port `5432` for PostgreSQL and ports `9092`, `9094`, and `9096` for Kafka.
+
+    After that, test the connection from the bastion host to the Kafka cluster or PostgreSQL instance.
+
+    ```bash
+    telnet <KAFKA_BROKER_HOST> <KAFKA_BROKER_PORT>
+    telnet <POSTGRES_HOST> <POSTGRES_PORT>
+    ```
+
+1. Verify that you can create SSH tunnels.
+
+    ```bash
+    # Command for Linux
+    ssh -L 9092:kafka-broker:9092 <SSH_BASTION_USER>@<SSH_BASTION_HOST>
+    ```
+
+    If you are unable to start the tunnel, enable `AllowTcpForwarding` and `PermitTunnel` in the SSH config file.
```

Comment:
If I recall correctly, creating the tunnel (i.e. starting the tunnel on the local machine) is always successful. A configuration issue only becomes visible if a connection is established through the tunnel (which can be tested with telnet, similar to above but connecting to localhost).